### PR TITLE
IAC-797 Improved default content.yaml archetypes for vRA

### PR DESCRIPTION
**Pull request by imihov1988**
…h/pull

### Checklist

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added relevant error handling and logging messages to help troubleshooting
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have updated CHANGELOG.md with a short summary of the changes introduced
- [x] I have tested against live environment, if applicable
- [ ] I have added relevant usage information (As-built)
- [ ] Any structure and/or content vRA-NG improvements are synchronized with vra-ng and ts-vra-ng archetypes
- [ ] Every new or updated Installer property is documented in docs/archive/doc/markdown/use-bundle-installer.md
- [ ] Dependencies in pom.xml are up-to-date
- [ ] My changes have been rebased and squashed to the minimal number of relevant commits
- [ ] My changes have a descriptive commit message with a short title, including a `Fixed #XXX -` or `Closed #XXX -` prefix to auto-close the issue

### CODE REVIEW

File: maven/archetypes/ts-vra-ng/src/main/resources/archetype-resources/vra/content.yaml [hunk 1]
```diff
@@ -27,7 +27,8 @@
 #     - policy name 2
 
 blueprint:
-subscription:
+  - Volume
```
Comment (Michaelpalacce) [inline maven/archetypes/ts-vra-ng/src/main/resources/archetype-resources/vra/content.yaml:30]: lets set this to [] as well?

**Review by Michaelpalacce — COMMENTED** [review]
Changelog + release.md

File: maven/archetypes/ts-vra-ng/src/main/resources/archetype-resources/vra/content.yaml [hunk 2]
```diff
@@ -37,11 +38,15 @@ region-mapping:
...
+catalog-item: []
+custom-resource: []
+resource-action: []
+catalog-entitlement: []
+
+property-group: []
 policy:
-  content-sharing:
+  content-sharing: []
+  lease: []
```
[the remark this answers is not in the document]
Comment (imihov1988) [reply maven/archetypes/ts-vra-ng/src/main/resources/archetype-resources/vra/content.yaml]: @ivo-kotev Ok, I removed them.

**Review by joroaf — APPROVED** [review]

**Review by Michaelpalacce — CHANGES_REQUESTED** [review]
Some minor changes needed

File: maven/archetypes/ts-vra-ng/src/main/resources/archetype-resources/vra/content.yaml (hunk 1 above)
Comment (imihov1988) [reply maven/archetypes/ts-vra-ng/src/main/resources/archetype-resources/vra/content.yaml:30]: @Michaelpalacce it doesn't work with empty blueprints as we have tested with @joroaf the newly created project from the archetype

Comment (Michaelpalacce) [reply maven/archetypes/ts-vra-ng/src/main/resources/archetype-resources/vra/content.yaml:30]: @imihov1988 You get an error when pulling empty blueprints? really?

**Review by Michaelpalacce — CHANGES_REQUESTED** [review]
Changelog + Release.md please

File: maven/archetypes/ts-vra-ng/src/main/resources/archetype-resources/vra/content.yaml (hunk 1 above)
Comment (imihov1988) [reply maven/archetypes/ts-vra-ng/src/main/resources/archetype-resources/vra/content.yaml:30]: @Michaelpalacce  Yes, it was not erroring out when trying to build it.

**Review by VenelinBakalov — APPROVED** [review]

**Pull request merged**